### ISSUE TITLE
"__test" folder not created post-build

### ISSUE THREAD

**Issue by szlatkow** [bchavez/Bogus#337]
Hi!

I followed the build instructions in the documentation locally and encountered an issue.

According to the instructions, if you follow the steps, you should see three folders "__compile", "__package" and "__test".

"__compile" and "__package" appear properly with the appropriate files inside, but the "__test" folder does not.

I looked through the *.fsx files used for the build and as far as I could understand, I couldn't see functionality for the creation of the "__test" folder.

The basis is declared in the Utils.fsx file for all the folders, but in the build.fsx, I couldn't find references to the declaration of the "__test" folder.

**Comment by bchavez** [comment]
Hi @szlatkow,

I don't think the `__test` folder is used anymore. IIRC, in the early days of **.NET Core** and AppVeyor, the only way I could get tests to show correctly in the AppVeyor UI was to dump the unit test results XML to a `__test` folder; then upload those XML results into AppVeyor.

That old process isn't used anymore because the `dotnet test` Test Platform is used now. Currently, the way tests are uploaded into AppVeyor is by using "special" `*.TestLogger` plugins such as:
https://github.com/bchavez/Bogus/blob/a6f94e3bb4f27d7c183d7730bca1489eafee5243/Source/Bogus.Tests/Bogus.Tests.csproj#L9

I suspect the references to `__test` are remnants of the early .NET Core days.

If you want to run tests,
* `build.cmd test` should do it.

Test results should get displayed (but not logged to disk). For example,

```
Microsoft (R) Test Execution Command Line Tool Version 16.8.0
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...
A total of 1 test files matched the specified pattern.
[xUnit.net 00:00:02.93]     Bogus.Tests.Rant_Generator.generate_product_reviews [SKIP]
  Skipped Bogus.Tests.Rant_Generator.generate_product_reviews [1 ms]
[xUnit.net 00:00:09.64]     Bogus.Tests.GitHubIssues.Issue121.Test [SKIP]
  Skipped Bogus.Tests.GitHubIssues.Issue121.Test [1 ms]
[xUnit.net 00:00:11.34]     Bogus.Tests.DataSetTests.ImageTest.DownloadAllTest [SKIP]
  Skipped Bogus.Tests.DataSetTests.ImageTest.DownloadAllTest [1 ms]
[xUnit.net 00:00:11.50]     Bogus.Tests.DataSetTests.InternetTests.can_get_avatar [FAIL]
  Failed Bogus.Tests.DataSetTests.InternetTests.can_get_avatar [335 ms]
```

If you really want the test results XML file, you'll have to follow these instructions in this repository for the `XunitXml.TestLogger`:
https://github.com/spekt/xunit.testlogger#usage

https://github.com/bchavez/Bogus/blob/a6f94e3bb4f27d7c183d7730bca1489eafee5243/Source/Bogus.Tests/Bogus.Tests.csproj#L20

```
cd Bogus\Source\Bogus.Tests
❯ dotnet test --logger:"xunit;LogFilePath=test_result.xml"
```

And that should get you a test XML file located in `Bogus\Source\Bogus.Tests\test_result.xml`

Essentially, in the early days of .NET Core, this was the command that was used to dump the `test_result.xml` file to the `__test` folder.

I hope this helps.

Thanks,
Brian

**Comment by bchavez** [comment]
Hi @szlatkow, the latest commit in `master` should create `__test\results.xml` file now. Fixed in 5219e22.
